purchase_work_acceptance/models/purchase.py

```diff
@@ -84,6 +87,13 @@ def action_create_invoice(self):
             }
         return super().action_create_invoice()
 
+    def _compute_invisible_create_wa(self):
+        for order in self:
+            invisible_create_wa = False
+            if any([not qty for qty in order.order_line.mapped("qty_to_accept")]):
```

Comment:
What if qty is negative?

Reply:
I think following is a bit easier to read (try not to use double negate)
```
hide_create_wa = True
for order in self:
    # Find lines with qty_to_accept > 0
    if order.order_line.filtered(lambda l: l.qty_to_accept > 0):
        hide_create_wa = False
    order.hide_create_wa = hide_create_wa
```

Reply:
fixed